build.gradle

```diff
@@ -8,10 +8,10 @@ buildscript {
     ]
 
     ext.versions = [
-            "kotlin"    : "1.6.21",
-            "compose"   : "1.0.2",
+            "kotlin"    : "1.8.21",
+            "compose"   : "1.4.7",
```

Comment:
Ah gotcha - it's hard to tell sometimes what in the build.gradle is for us vs what merchants would see/pull in 😅

Reply:
Yeah Gradle has a convention where there's a top-level `build.gradle` and a separate `build.gradle` for each module. There's a few other ways to build a gradle project, but the way we have it is fairly standard for Android Studio projects. But it is good to callout out these details because a change in the top-level build.gradle could have an effect on the feature modules we ship.